### DISCUSSION TITLE
Grouping of list items in PolyLookup

### DISCUSSION THREAD

**Discussion by wenstarner** [khoait/DCE.PCF#143] [Ideas]
Hello, 

Just another idea I thought of, that I was thinking might be a great feature for your PolyLookup!

Would it be possible to enable **grouping** of list values in PolyLookup? Like in the screenshot at the bottom of this post (taken from your LookDown component!).

Grouping could be great for organising a long list of items... exactly how grouping is useful in the LookDown component.

I was also thinking of how it could be really useful for tagging multiple Category/Subcategory combinations to a record. The user would need to only select a Subcategory value, which would appear under a group for the Category. The integrator/developer could then implement plugin functionality to write the Category value, based on the Subcategory selected.

I guess my thinking is: What if LookDown, with its nice grouping of list values, allowed for selection of multiple items?

![image](https://github.com/user-attachments/assets/f1be9dbe-4df3-4311-aae5-7d8c8902d8b8)

**Comment by khoait** [comment]
@wenstarner thanks for your idea! I agree this can be useful, however, it could cause some confusion as we have searching and paging (max number of items limit in dropdown) functionality in the PolyLookup. The number of items in a group shown in the dropdown could be less than the actual number because of search or paging, and users are not aware of this thinking all items are shown in the group.

Let me sleep on it and get back.

**Reply by wenstarner** [reply]
Ah, that’s a great point. Hopefully there might be a method. Thank you for your response! Very much appreciated, as always :)
